clients/java/src/main/java/io/camunda/client/impl/CamundaClientImpl.java

```diff
@@ -404,14 +404,15 @@ public static ManagedChannel buildChannel(final CamundaClientConfiguration confi
     final URI address;
     address = config.getGrpcAddress();
 
-    final NettyChannelBuilder channelBuilder =
-        NettyChannelBuilder.forAddress(address.getHost(), address.getPort());
+    final String target = String.format("dns:///%s:%d", address.getHost(), address.getPort());
```

Comment:
@entangled90 what is the semantic of this "dns:///%s:%d" URL? Do you still just provide the DNS name of the GW service?